lib/common_test/src/ct_groups.erl

```diff
@@ -66,6 +66,17 @@ find_groups1(Mod, GrNames, TCs, GroupDefs) ->
 		 GroupDefs, FindAll),
     [Conf || Conf <- Found, Conf /= 'NOMATCH'].
 
+%% Finds out if the given group, or child cases / child group of a group, is non-empty.
+not_empty_conf({conf, _Props, _Init, Children, _End}) -> not_empty_conf(Children);
+not_empty_conf([] = _Cases) -> false;
+not_empty_conf([_ | _] = _Cases) -> true;
+not_empty_conf(Name) when is_atom(Name) -> true;
+% Other, unexpected formats we don't actively care about, such as `{mnesia_evil_backup, all}`.
+not_empty_conf(_Format) -> true.
+
+elide_empty_confs(Confs) ->
```

Comment:
Thank you for the suggestion, I just renamed it to `remove_empty_confs`. Since it can also be something else than a group and is tagged with `conf`, hence I chose `conf` instead of `groups`.